File: Solutions/1897. Redistribute Characters to Make All Strings Equal/1897. Redistribute Characters to Make All Strings Equal.cpp
class Solution {
public:
    bool makeEqual(vector<string>& words) {
        int cnt[26]={};
        for(const auto& w:words){
          for(const auto& c:w)
              ++cnt[c-'a'];
        }
        for(auto x:cnt)
            if(x%words.size())
                return false;
        return true;
    }
};

File: Solutions/1897. Redistribute Characters to Make All Strings Equal/1897. Redistribute Characters to Make All Strings Equal.java
class Solution {
    public boolean makeEqual(String[] words) {
        int[] cnt=new int[26];
        for(var w:words)
            for(var c:w.toCharArray())
                ++cnt[c-'a'];
        for(var x:cnt)
            if(x%words.length!=0)
                return false;
        return true;
    }
}